test/unit/puppetclass_test.rb

```diff
@@ -246,4 +246,32 @@ def check_object_config_group(obj)
       end
     end
   end
+
+  context "going down the rabbit hole" do
```

Comment:
:laughing: , but :-1:  , I think it's better if we're concise in our code, it saves time when reading it